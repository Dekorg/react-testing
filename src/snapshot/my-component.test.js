import React from 'react';
import renderer from 'react-test-renderer';

import MyComponent from './my-component';

it('should be rendered correctly', () => {
    const component = renderer.create(
        <MyComponent title="some title">some content</MyComponent>
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});