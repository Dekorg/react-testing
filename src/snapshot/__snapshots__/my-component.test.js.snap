exports[`test should be rendered correctly 1`] = `
<section>
  <header>
    some title
  </header>
  <p>
    some content
  </p>
</section>
`;
